scala/README.md

```diff
@@ -17,37 +17,37 @@
 
 ### Data Structures:
 - Stack
-  - push(x): Pushes element x onto stack.  
-  - pop(): Removes the element on top of the stack.  
-  - top(): Gets the top element.  
-  - empty(): Returns whether the stack is empty.  
+  - push(x): Pushes element x onto stack.
```

Comment:
Looks like my VIM made unexpected changes? :thinking:

Reply:
:joy:  I thought you made this data structure only push-able.

Reply:
In a sense it is, you can't remove an element from a Bloom Filter

Reply:
Oh, it removed trailing spaces. Cool